website/models.py

```diff
@@ -61,3 +61,31 @@ def createUserProfile(sender, instance, created, **kwargs):
 @receiver(post_save, sender=User)
 def saveUserProfile(sender, instance, **kwargs):
     instance.profile.save()
+
+
+@receiver(post_delete, sender=Profile)
+def deleteOnDelete(sender, instance, **kwargs):
+    if instance.photo:
+        if os.path.basename(instance.photo.name) != "defaultuser.png":
+            if os.path.isfile(instance.photo.path):
+                os.remove(instance.photo.path)
```

Comment:
I'm thinking there is some possibilities of the remove causing exceptions. We should probably try/catch them and silently ignore them.

Reply:
What exceptions are you thinking of?